packages/reactivity/__tests__/effectScope.spec.ts

```diff
@@ -192,6 +192,26 @@ describe('reactivity/effect/scope', () => {
     expect(dummy).toBe(7)
   })
 
+  it('should warn onDispose() is called when there is no active effect scope', () => {
+    let dummy = 0
+    const scope = new EffectScope()
+    scope.run(() => {
+      onScopeDispose(() => (dummy += 1))
+      onScopeDispose(() => (dummy += 2))
+    })
+
+    expect(dummy).toBe(0)
```

Comment:
You should use spies to count how often a function is called 🙂 

```suggestion
    const spy = jest.fn()
    const scope = new EffectScope()
    scope.run(() => {
      onScopeDispose(spy)
    })

    expect(spy).toHaveBeenCalledTimes(0)
```

Reply:
has to fix!Thank you for your review.